alarm-frontend/src/App.js

```diff
@@ -1,22 +1,43 @@
-import AddAlarm from "./Components/AddAlarm";
+import React, { useState } from "react";
 import Clock from "./Components/Clock";
 import ContextAlarm from "./Components/Context";
 import AlarmList from "./Components/AlarmList";
+import Math from "./Components/Math";
 
 function App() {
-  return (
-    <section>
-      <div>
+  const [type, setCurrentType] = useState("Base");
+  const [difficulty, setCurrentDiff] = useState();
```

Comment:
Try to follow this pattern so you can easily search for the corresponding callback later.
`const [stateSliceName, setStateSliceName] = useState();`

In this case i'd change `setCurrentDiff` => `setDifficulty`